modules/loki-rbac-permissions.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rbac-permissions_{context}"]
+= Authorizing Loki rules RBAC permissions
+
+// May need to re-add this after 5.8 release - check with eng later
+// In logging 5.7 and later, the Cluster Logging Operator provides `alertingrule-editor-role` and `recordingrule-editor-role` cluster roles, which enable users to modify alerting and recording rules for the LokiStack.
+
+You can authorize RBAC permissions for modifying rules by binding the required cluster roles to a username.
```

Comment:
Just to clarify that as of now it's expected that they create their own set of ClusterRoles
```suggestion
To allow users to create and manage their own rules, a cluster admin needs to create a ClusterRole that will give the necessary RBAC permissions and then bind that role to the required usernames
```

Reply:
Phrased slightly differently, feel free to pick the one you prefer the most :) 
```suggestion
Cluster admins can give RBAC permissions for users to create and manage their own rules by creating a ClusterRole's and then binding those roles to the required usernames.
```

Reply:
Sounds good. I'm also adding a link to https://docs.openshift.com/container-platform/4.13/authentication/using-rbac.html in the additional resources, because I think this would be useful here?